docs/contents.md

```diff
@@ -5,7 +5,8 @@
 
 ## Using Glint
 
-- [Configuration](configuration.md)
+- [Configuration](./configuration/_index_.md)
```

Comment:
I'm not sure this matches the file name?

Reply:
You are correct; it absolutely did not! d140f11